src/main/webapp/resources/js/pages/user/components/UserAccountNav.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Menu } from "antd";
+
+/**
+ * React component to display the user account navigation.
+ * @returns {*}
+ * @constructor
+ */
+export default function UserAccountNav() {
+  const lastElement = window.location.pathname.split("/").pop();
```

Comment:
Better to use location provided by the router: 

```
const location = useLocation();
location.pathname ...
```

Reply:
Please see 30e8f58f67a844dcc7566b6ca3c7d5b0cdb2bb4a.